src/app/sections/ConnectWallet.tsx

```diff
@@ -34,23 +34,25 @@ const ConnectWallet = ({ renderButton }: Props) => {
   )
 
   const list = [
-    ...availableConnections
-      .filter(({ type }) => type !== "READONLY")
-      .map(({ type, identifier, name, icon }) => ({
+    ...availableWallets
+      .filter(({ isInstalled }) => isInstalled)
+      .map(({ id, name, icon }) => ({
         src: icon,
         children: name,
-        onClick: () => connect(type, identifier),
+        onClick: () => connect(id),
       })),
     {
       icon: <UsbIcon />,
       to: "/auth/ledger",
       children: t("Access with ledger"),
     },
-    ...availableInstallations.map(({ name, icon, url }) => ({
-      src: icon,
-      children: t(`Install ${name}`),
-      href: url,
-    })),
+    ...availableWallets
+      .filter(({ isInstalled, website }) => !isInstalled && website)
+      .map(({ name, icon, website }) => ({
+        src: icon,
+        children: t(`Install ${name}`),
+        href: website ?? "",
+      })),
```

Comment:
Current:

<img width="623" alt="image" src="https://github.com/terra-money/station/assets/30275692/3eb345f6-4413-4253-8236-d94233e4a93b">

On Station Web:

https://github.com/terra-money/station/assets/30275692/cfe8b188-f24f-4343-9e9f-00dae8049f1c

Reply:
Most of the available extensions that we are currently showing don't even work with station web, and if they do they don't have interchain functionalities.

They can make a PR to wallet-kit if they want to fix their integration.

Reply:
@alecande11 ok.  And is there a reason we only have 2 staking assets on Mainnet if we are connected with a ledger?

<img width="358" alt="image" src="https://github.com/terra-money/station/assets/30275692/e5036b1e-72db-4481-bf31-157b58c872dc">

Reply:
I think it's because you imported only your 330 (Terra) account and not the 118 (using the cosmos app)

Reply:
@terran6 which install wallet option?

Reply:
> 

@alecande11 The options in the video I posted which are in Station, but not this branch.

Reply:
Oh, that code was on wallet-provider, it's already removed and replaced with wallet-kit

Reply:
Nvm @terran6  I see what you mean, we still need the code that shows the install link since we are still using it.
If a user doesn't have the extension installed we want to redirect them to the installation page.